app/src/main/java/wilsonoh/sagyo/tasks/TodoTask.java

```diff
@@ -0,0 +1,19 @@
+package wilsonoh.sagyo.tasks;
+
+import wilsonoh.sagyo.commands.CommandType;
+
+public class TodoTask extends Task {
+
+    public TodoTask(String taskName) {
+        super(taskName);
+    }
+    @Override
```

Comment:
An indentation could be left after this closing bracket to follow consistency.

Reply:
Good catch! I'll add a clang-format rule for this.